### PR TITLE
Bring back the sword

### PR DESCRIPTION
**Pull request by rorhug**

### CODE REVIEW

File: src/pages/index.astro [hunk 1]
```diff
@@ -10,6 +10,11 @@
 		<meta name="generator" content={Astro.generator} />
 		<link href="https://fonts.googleapis.com/css2?family=Berkshire+Swash&display=swap" rel="stylesheet">
 		<title>é/acc</title>
+		<style>
+			body {
+				cursor: url('Zaros godsword.png'), auto;
```
Comment (stephenbyrne99) [inline src/pages/index.astro]: can we use armadyl

Comment (stephenbyrne99) [reply src/pages/index.astro]: https://oldschool.runescape.wiki/images/thumb/Armadyl_godsword_detail.png/260px-Armadyl_godsword_detail.png?f2566

Comment (stephenbyrne99) [reply src/pages/index.astro]: pk VID: https://www.youtube.com/watch?v=gI_-vJmPoRY

**Comment by robertmtolan** [comment]
Hey guys, looks great so far! Very excited to see this go online, let me know if you want a helping hand etc - hoping to build up my commits outside of work stuff

**Pull request merged**